.github/workflows/pr.yaml

```diff
@@ -22,6 +22,14 @@ jobs:
         run: cargo jjs-check --no-default --rustfmt
       - name: run shellcheck
         run: cargo jjs-check --no-default --shellcheck
+  misspell-fixer:
+    name: misspell-fixer
+    runs-on: "ubuntu-18.04"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: sobolevn/misspell-fixer-action@master
+        with:
+          options: '-svuRVD .'
```

Comment:
Please, change this flags to long name. `svuRVD` is difficult for understanding.

Reply:
It turned out long names are not supported